src/App.js

```diff
@@ -9,12 +13,21 @@ import { useTheme } from './hooks';
 const App = () => {
   const [theme, changeTheme] = useTheme();
   return (
-    <ThemeProvider theme={Theme[theme]}>
-      <GlobalStyles />
-      <Toggle type={theme} toggleTheme={changeTheme}>
-        Toggle Theme
-      </Toggle>
-    </ThemeProvider>
+    <div>
+      <ThemeProvider theme={Theme[theme]}>
+        <GlobalStyles />
+        <Switch>
```

Comment:
I think that routes should be in one place. Currently I can see these in App.js and TopApp.js. Better, think about moving all routes declaration to one file.

Reply:
Top app is a wrapper which ideally divides the very root level functionalities. However, let me think through it.